static/js/controllers/contacts.js

```diff
@@ -29,6 +30,20 @@ var _ = require('underscore'),
       $scope.selected = null;
       $scope.filters = {};
 
+      var defaultTypeFilter = {};
+      function setDefaultTypeFilter() {
+        return UserSettings().then(function(u) {
+          return u.facility_id && DB().get(u.facility_id).then(function(facility) {
+            var targetType = CONTACT_TYPES[CONTACT_TYPES.indexOf(facility.type) + 1];
```

Comment:
So, I'm doing something similar for my issue, finding the level below a given doc.
We have `CONTACT_TYPES` constant, and [ContactSchema service](https://github.com/medic/medic-webapp/blob/master/static/js/services/contact-schema.js), and I think we're just crossing fingers that they're the same. 
I've been using ContactSchema, looks like there's more stuff in it.
Thoughts?